content/en/docs/concepts/workloads/pods/ephemeral-containers.md

```diff
@@ -81,134 +81,83 @@ you can view processes in other containers.
-enabled, and Kubernetes client and server version v1.16 or later.
+The examples in this section require Kubernetes server components of v1.16
+or later with the `EphemeralContainers` [feature gate](
+/docs/reference/command-line-tools-reference/feature-gates/) enabled and
+`kubectl` version v1.18 or later.
 {{< /note >}}
 
-The examples in this section demonstrate how ephemeral containers appear in
-the API. You would normally use a `kubectl` plugin for troubleshooting that
-automates these steps.
-
-Ephemeral containers are created using the `ephemeralcontainers` subresource
-of Pod, which can be demonstrated using `kubectl --raw`. First describe
-the ephemeral container to add as an `EphemeralContainers` list:
-
-```json
-{
-    "apiVersion": "v1",
-    "kind": "EphemeralContainers",
-    "metadata": {
-            "name": "example-pod"
-    },
-    "ephemeralContainers": [{
-        "command": [
-            "sh"
-        ],
-        "image": "busybox",
-        "imagePullPolicy": "IfNotPresent",
-        "name": "debugger",
-        "stdin": true,
-        "tty": true,
-        "terminationMessagePolicy": "File"
-    }]
-}
+The `kubect alpha debug` command supports adding ephemeral containers to a
+running Pod. The `pause` container image makes a good example because it is
+built from `SCRATCH`.
+
+```shell
+kubectl run pause --image=k8s.gcr.io/pause:3.1 --restart=Never
+kubectl 
 ```
 
-To update the ephemeral containers of the already running `example-pod`:
+If you attempt to use `kubectl exec` to create a shell you will see an error
+because there is no shell in this container image.
 
 ```shell
-kubectl replace --raw /api/v1/namespaces/default/pods/example-pod/ephemeralcontainers  -f ec.json
+kubectl exec -it pause -- sh
 ```
 
-This will return the new list of ephemeral containers:
-
-```json
-{
-   "kind":"EphemeralContainers",
-   "apiVersion":"v1",
-   "metadata":{
-      "name":"example-pod",
-      "namespace":"default",
-      "selfLink":"/api/v1/namespaces/default/pods/example-pod/ephemeralcontainers",
-      "uid":"a14a6d9b-62f2-4119-9d8e-e2ed6bc3a47c",
-      "resourceVersion":"15886",
-      "creationTimestamp":"2019-08-29T06:41:42Z"
-   },
-   "ephemeralContainers":[
-      {
-         "name":"debugger",
-         "image":"busybox",
-         "command":[
-            "sh"
-         ],
-         "resources":{
-
-         },
-         "terminationMessagePolicy":"File",
-         "imagePullPolicy":"IfNotPresent",
-         "stdin":true,
-         "tty":true
-      }
-   ]
-}
 ```
+OCI runtime exec failed: exec failed: container_linux.go:346: starting container process caused "exec: \"sh\": executable file not found in $PATH": unknown
+```
+
+You can instead add a debugging container using `kubectl alpha debug`.
+
+```shell
+kubectl alpha debug -it pause --image=busybox --target=pause
+```
+
+```
+Defaulting debug container name to debugger-8xzrl.
+If you don't see a command prompt, try pressing enter.
+/ # ps auxww
```

Comment:
This doesn't match the style guide's recommendations. Instead, [separate commands from their output](https://kubernetes.io/docs/contribute/style/style-guide/#separate-commands-from-output)